### PR TITLE
Add basic content to portfolio

### PR DESCRIPTION
**Pull request by eash130**
It doesn't really follow the general content of a portfolio, but I will use it to hopefully make something cool.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,10 +9,34 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Hello! My name is Eashan.</p>
+      <p>I'm a CS major at the Georgia Institute of Technology and I have a passion for social good. In an constantly technologizing society, there is only so much we can do to help others.
```
Comment (alygerharter) [inline portfolio/src/main/webapp/index.html]: are you able to fix the formatting so these lines aren't so long?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -9,10 +9,34 @@
...
+      <p>In my free time, I play a lot of video games, especially since I have been in quarantine. Some of the games I play/games I enjoy playing include:</p>
+      <ul id="game-list">
+          <li>Apex Legends</li>
+          <li>Call of Duty</li>
+          <li>Fortnite</li>
+          <li>Minecraft</li>
+          <li>Town of Salem</li>
+      </ul>
+      <p>Click here to get a random iconic quote from Spongebob:</p>
+      <button onclick="addRandomGreeting()">*cue Spongebob transition*</button>
```
Comment (alygerharter) [inline portfolio/src/main/webapp/index.html]: does this work? if not, maybe save for a future PR

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -9,10 +9,34 @@
...
       <div id="greeting-container"></div>
```
Comment (alygerharter) [inline portfolio/src/main/webapp/index.html]: is this used?

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -9,10 +9,34 @@
...
+      <br/>
+      <p>Here are some great quotes</p>
+      <ul id="quote-list">
```
Comment (alygerharter) [inline portfolio/src/main/webapp/index.html]: since this is empty, save it for a future PR

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (eash130) [reply portfolio/src/main/webapp/index.html]: Fixed

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (eash130) [reply portfolio/src/main/webapp/index.html]: Yes, new commit has the corresponding JS file.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (eash130) [reply portfolio/src/main/webapp/index.html]: Fixed

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -15,14 +15,20 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const quotes =
+      ['I\'m ready!', 'The best time to wear a striped sweater is all the time.',
```
Comment (alygerharter) [inline portfolio/src/main/webapp/script.js]: put each item on a separate line so they are easier to read

Comment (eash130) [reply portfolio/src/main/webapp/script.js]: Fixed

**Review by alygerharter — APPROVED** [review]

**Pull request merged**